Error: Cannot find module './js/bundledPlugins/previouslyUsedKeywords'

## What did you expect to happen?
Me to run the app provided in the README

## What happened instead?
I got an error: Error:` Cannot find module './js/bundledPlugins/previouslyUsedKeywords'`


## How can we reproduce this behavior?
2246  sudo npm init
2248  sudo npm install https://github.com/Yoast/YoastSEO.js#develop
2252  sudo nano index.js
2253  node index.js

--- a/README.md
+++ b/README.md
@@ -16,7 +16,7 @@ Also included is a preview of the Google search results which can be assessed us
 You can install YoastSEO.js using npm:
 
 ```bash
-npm install https://github.com/Yoast/YoastSEO.js#develop
+npm install yoastseo
 ```
 
 ## Usage

